bun.lock

```diff
@@ -1,5 +1,6 @@
 {
   "lockfileVersion": 1,
+  "configVersion": 0,
```

Comment:
you might want to update your version of bun. 

this new config is related to [isolated installs](https://bun.com/docs/pm/isolated-installs#isolated-installs) and (i believe) backwards compatible with older versions of bun too.